jujugui/static/gui/src/app/components/usso-login-link/_usso-login-link.scss

```diff
@@ -0,0 +1,26 @@
+.usso-login {
+  position: relative;
+
+  &__notification {
+    background: #fff;
+    box-shadow: 0 1px 4.85px 0.15px rgba(0,0,0,0.2);
+    box-sizing: border-box;
+    color: #666;
+    font-size: 14px;
+    line-height: normal;
+    margin: 10px auto 0;
+    padding: 12px 25px;
+    position: absolute;
+    z-index: index($z-indexed-elements, usso-login-link);
+    top: 100%;
+    right: 10px;
+    transition: opacity 0.3s;
+    width: 300px;
+    display: none;
+  }
+
+  a:hover + &__notification,
+  button:hover + &__notification {
```

Comment:
Did the link or button not have classes? It is much better to target classes in components so mark up can change and it still work. For example, later someone could change the link to a span with an onClick function and would take time to work it out.